projects/cashmere-examples/src/lib/button-primary/button-primary-example.component.scss

```diff
@@ -0,0 +1,3 @@
+button {
+    margin: 10px;
```

Comment:
This seems to be a very common thing to add to each of the examples. What was the reason to add this to so many style sheets? Could it be something that should be pushed to a more global style sheet?

Reply:
We could promote it to `projects\cashmere-examples\src\project-template\src\styles.scss`...

I just want to remove as much smoke and mirrors as I can for the examples.  As Cashmere buttons don't have margin, but you almost never use them without margin, users may assume after looking at the examples that the margins are a part of Cashmere buttons... but they are not.  I don't think it hurts to put this in the example stylesheet if you will have to do this in order to achieve the style displayed.